workspaces/src/rpc/api.rs

```diff
@@ -48,20 +47,19 @@ impl From<FinalExecutionOutcomeView> for CallExecutionResult {
     }
 }
 
-pub async fn display_account_info(account_id: AccountId) -> Result<AccountInfo, String> {
-    let query_resp = tool::json_client()
+pub async fn display_account_info(account_id: AccountId) -> anyhow::Result<AccountInfo> {
+    let query_resp = client::new()
         .call(&RpcQueryRequest {
             block_reference: Finality::Final.into(),
             request: QueryRequest::ViewAccount {
                 account_id: account_id.clone(),
             },
         })
-        .await
-        .map_err(|err| err.to_string())?;
+        .await?;
 
     let account_view = match query_resp.kind {
         QueryResponseKind::ViewAccount(result) => result,
-        _ => return Err("Error call result".to_owned()),
+        _ => return Err(anyhow!("Error call result")),
```

Comment:
doesn't need to change since this can be more readable, but can use `bail!` for convenience https://docs.rs/anyhow/1.0.48/anyhow/macro.bail.html

Reply:
ahh, didn't know about this macro. Will change it to `bail`. thanks!